src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,222 @@
+                      </Box>
+                    </Box>
+                  );
+                }}
+              </ZUIFuture>
+            )}
+          </IconButton>
+        </DrawerHeader>
+        <Divider />
+        <List>
+          {menuItemsMap.map((item) => (
+            <ListItem
+              key={item.name}
+              disableGutters
+              disablePadding
+              sx={{ display: 'block' }}
+            >
+              <NextLink
+                href={
+                  item.name === 'home'
+                    ? '/organize/'
+                    : `/organize/${orgId}/${item.name}`
+                }
+                passHref
               >
-                <Event />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-        </Box>
-      </List>
+                <IconButton
+                  color={
+                    key.startsWith('/' + item.name) ? 'primary' : 'secondary'
+                  }
+                  sx={{
+                    justifyContent: open ? 'initial' : 'center',
+                    minHeight: 48,
+                    px: 2.5,
+                  }}
+                >
+                  <ListItemIcon
+                    sx={{
+                      justifyContent: 'center',
+                      minWidth: 0,
+                      mr: open ? 3 : 'auto',
+                    }}
+                  />
+                  {item.icon}
+                  <ListItemText
+                    primary={messages.organizeSidebar[item.name]()}
+                    sx={{ marginLeft: 3, opacity: open ? 1 : 0 }}
+                  />
+                </IconButton>
+              </NextLink>
+            </ListItem>
+          ))}
+        </List>
+      </Drawer>
     </Box>
   );
 
-  return (
-    <div className={classes.root}>
-      <AppBar className={classes.appBar} position="fixed">
-        <Toolbar>
-          <IconButton
-            aria-label="open drawer"
-            className={classes.menuButton}
-            color="inherit"
-            edge="start"
-            onClick={handleDrawerToggle}
-            size="large"
-          >
-            <Menu data-test="menu-button" />
-          </IconButton>
-        </Toolbar>
-      </AppBar>
-      <nav aria-label="mailbox folders" className={classes.drawer}>
-        <Drawer
-          anchor={theme.direction === 'rtl' ? 'right' : 'left'}
-          classes={{
-            paper: classes.drawerPaper,
-          }}
-          ModalProps={{
-            keepMounted: true, // Better open performance on mobile.
-          }}
-          onClose={handleDrawerToggle}
-          open={mobileOpen}
-          variant="temporary"
-        >
-          {drawer}
-        </Drawer>
-
-        <Drawer
-          classes={{
-            paper: classes.drawerPaper,
-          }}
-          open
-          variant="permanent"
-        >
-          {drawer}
-        </Drawer>
-      </nav>
-    </div>
-  );
+  return OrganizeSideBar;
```

Comment:
Why create a variable instead of just returning right away?

Reply:
I got the variable because we were having 2 drawers (mobile and desktop) before so we were rendering the variable inside every drawer. We don't need that anymore so I will use the return 💯